### ISSUE TITLE
[Linux]Graphical Flickering in Menu and Game since last update

### ISSUE THREAD

**Issue by 420git** [ValveSoftware/Dota-2#1846]
It never happened before It started appearing aftr the update which removed the compendium, random boxes of different colors started appearing and change with mousse position.

OS:Arch
CPU: Intel(R) Core(TM) i5-3230M CPU @ 2.60GHz
VGA:AMD Radeon 6610M Drivers: Mesa10.6.5

**Comment by danslo** [comment]
Same here.

**Comment by 420git** [comment]
I decided to stop using the outdated and buggy steam runtime and just get the required drivers myself and it works now.
To use steam without its runtime use this to start steam
$ STEAM_RUNTIME=0 steam
or edit the shortcut of steam to this 
STEAM_RUNTIME=0 /usr/bin/steam %U
ofcourse you will first be faced with a message mentioning missing required drivers .Ive tracked and installed most of them, but I think the easiest way to do this which I finally had to do was add the alucard repository if you are using Arch , and install the steam-libs group this made me run steam without the runtime and without any missing libs.

**Comment by 420git** [comment]
lol i deleted my comment by mistake if you got it in your mail please post it again here so others can know about it.
tl dr ; I run steam without its original runtime after getting steam-libs from alucard repository

**Comment by danslo** [comment]
Your e-mail:

>I decided to stop using the outdated and buggy steam runtime and just get the required drivers myself and it works now.
>To use steam without its runtime use this to start steam
>$ STEAM_RUNTIME=0 steam
>or edit the shortcut of steam to this 
>STEAM_RUNTIME=0 /usr/bin/steam %U
>ofcourse you will first be faced with a message mentioning missing required drivers .Ive tracked and installed most of them, but I think the easiest way to do this which I finally had to do was add the alucard repository if you are using Arch , and install the steam-libs group this made me run steam without the runtime and without any missing libs.
